data/human/weapons.txt

```diff
@@ -11,6 +11,194 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Tethys Cannon"
+	category "Turrets"
+	cost 1067000
+	mass 128
+	thumbnail "outfit/rnd_gun2_turret"
+	"outfit space" -110
+	"weapon capacity" -97
+	"drag" 7
+	"required crew" 4
+	"turret mounts" -1
+	"shell capacity" 40
+	"active cooling" 4
+	"cooling energy" 3
```

Comment:
```suggestion

```
These weapons are already unique enough with their size and range, they don't need cooling attributes built-in as well. 
If you do want to go this route, have them built into their own outfit that is meant to be paired with the weapons, but let the player decide to have them or not.